### PR TITLE
fix: Player movement while hacking

### PR DESCRIPTION
**Pull request by OriolSors**
Fixed issues with player movement while hacking

### CODE REVIEW

**Review by markmeliaa — COMMENTED** [review]
I've tried the level 2 scene and while the movement while hacking seems to be fixed, once I get inside the hacking area, I can trigger the hacking with the E key even after leaving the hacking area and being far far away, is this a known bug?

File: Game/Scripts/PlayerHackingUseScript.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ void PlayerHackingUseScript::Start()
 	transform = GetOwner()->GetComponentInternal<ComponentTransform>();
 	rigidBody = GetOwner()->GetComponentInternal<ComponentRigidBody>();
 	hackZone = nullptr;
-
+	playerManager = GetOwner()->GetComponentInternal<PlayerManagerScript>();
```
Comment (markmeliaa) [inline Game/Scripts/PlayerHackingUseScript.cpp:34]: Forgot to add this, we should not use `GetComponentInternal` in the gameplay scripts, use `GetComponent` instead.

Comment (OriolSors) [reply Game/Scripts/PlayerHackingUseScript.cpp]: Done! Thanks for the advice!

**Comment by OriolSors** [comment]
> I've tried the level 2 scene and while the movement while hacking seems to be fixed, once I get inside the hacking area, I can trigger the hacking with the E key even after leaving the hacking area and being far far away, is this a known bug?

Yes, fixed. I didn't know that issue, first time noticed. Thank you again!!!

**Review by markmeliaa — APPROVED** [review]

**Review by Piterga12 — APPROVED** [review]
Good job! Works perfectly now!

**Pull request merged**